web/react/stores/user_store.jsx

```diff
@@ -3,7 +3,6 @@
 
 var AppDispatcher = require('../dispatcher/app_dispatcher.jsx');
 var EventEmitter = require('events').EventEmitter;
-var assign = require('object-assign');
```

Comment:
So it looks like we won't be able to remove "object-assign" completely from the project, since babelify uses it as a dependency, but we can change to that syntactic sugar.